ironfish/src/network/types.ts

```diff
@@ -20,6 +20,12 @@ export enum NetworkMessageType {
   SignalRequest = 12,
 }
 
+// Enums contain bi-directional mappings so filter out duplicate enum keys
+// This is used for some cases where we need to do some action for each message type
+export const NetworkMessageTypeList = Object.keys(NetworkMessageType)
+  .map((k) => parseInt(k))
+  .filter((k) => !isNaN(k))
```

Comment:
Do you want to move this into an IntEnumUtils similar to our StrEnumUtils? It would be nice if this code was reusable for IntEnums in case anyone else needed this too. https://github.com/iron-fish/ironfish/blob/master/ironfish/src/utils/enums.ts

Reply:
Yea that's a good idea. I'll do that